velox/expression/CastHooks.h

```diff
@@ -21,6 +21,10 @@
 
 namespace facebook::velox::exec {
 
+enum PolicyType { LegacyCastPolicy = 1, PrestoCastPolicy, SparkCastPolicy };
+
+fmt::underlying_t<PolicyType> format_as(PolicyType f);
```

Comment:
Where is this func being used?

Reply:
This is automatically called by fmt, if you want to print out a PolicyType ; fmt 9 and forward require you to have format_as(<Type>) defined for any custom enum's.